Automattic/wp-calypso
# A4A: Implement the 'Add site via A4A plugin' modal.

This pull request implements the **'Add site via A4A plugin'** modal in the Site selector dropdown menu. **Note: The pull request implements the modal based on the i1 design, which will likely change in i2.**

<img width="813" alt="Screenshot 2024-07-02 at 8 56 39 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/f8342c68-8b8f-428b-b5e8-7fa1e41ddd11">



Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/680

## Proposed Changes

* Implement the `A4AConnectionModal` component.

## Testing Instructions

* Use the A4A live link and go to `/overview` page.
* Click the 'Add site' dropdown menu.
* Click the 'Via A4A' option.
* Confirm the A4A modal appears.
* Confirm the links and copies are correct.

## Pre-merge Checklist
